packages/web3-circle-sdk/src/FaucetApi.ts

```diff
@@ -6,11 +6,9 @@ export class FaucetApi extends BaseApi {
    * Request testnet tokens for your wallet.
    * https://developers.circle.com/api-reference/w3s/programmable-wallets/request-testnet-tokens
    * @param params the parameters for the faucet request
-   * @returns a boolean that indicates if the faucet request was successful
+   * @returns void
```

Comment:
```suggestion
 * @returns Promise<void> - A promise that resolves when the request is complete.
```

Reply:
and same for other instances in this PR

Reply:
For consistency, this should read `@returns a promise that resolves when the request is complete`